eng/build.ps1

```diff
@@ -0,0 +1,27 @@
+[CmdletBinding(PositionalBinding=$false)]
+Param(
+  [string] $ArchGroup,
+  [switch] $Release,
+  [switch] $c,
+  [Parameter(ValueFromRemainingArguments=$true)][String[]]$ExtraArgs
+)
+
+$arguments = ""
+
+foreach ($argument in $PSBoundParameters.Keys)
+{
+  switch($argument)
+  {
+    "Debug"      { $arguments += "/p:ConfigurationGroup=Debug " }
+    "Release"    { $arguments += "/p:ConfigurationGroup=Release " }
+    "ExtraArgs"  { $arguments += $ExtraArgs }
+    "c"          { $arguments += "-rebuild " }
+    default      { $arguments += "/p:$argument=$($PSBoundParameters[$argument]) " }
+  }
+}
+
+#$arguments += " -warnaserror:`$false"
```

Comment:
Should we remove this comment?

Reply:
Yes. I have some of these cleaned up locally for my corefx PR, just haven't pushed them yet.